notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Can you mention that this is seconds? Otherwise it is not known to user until further down the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Sure, we have converted this to hours and mentioned that in the code comment.</p>